changelog/fragments/change-scImagePolicy.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      "Changed default image Pull Policy to Always for scorecard"
```

Comment:
@theishshah this change is not user facing (unless there is some prior PR that this follows up on), since the change was only made to testdata. If that is the case, can you submit a PR to remove this fragment?